test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -316,7 +316,7 @@ private static void PushChanges(RegressionTestResult testResult, string workingF
             }
             else
             {
-                Exec("git", $"{s_gitCmdAuth} push origin HEAD:{s_repositoryName}", cwd: workingFolder, secrets: s_gitCmdAuth);
+                Exec("git", $"{s_gitCmdAuth} push origin HEAD:{s_repositoryName}", cwd: workingFolder, allowExitCodes: new[] { 0 }, secrets: s_gitCmdAuth);
```

Comment:
please make `allowedExitCodes` default to `[0]`.

Reply:
Not sure if other weird situations may occur